config/config.go

```diff
@@ -597,6 +598,12 @@ type Debug struct {
 	OverrideToken       string              `mapstructure:"override_token"`
 }
 
+type Server struct {
+	ExternalUrl string `mapstructure:"external_url"`
+	GdprID      string `mapstructure:"gdpr_id"`
+	Datacenter  string `mapstructure:"datacenter"`
+}
```

Comment:
The `Server` struct should just be used for communicating values down to adapters. The External URL should be copied from `Configuration.ExternalURL` and the GDPR ID should be copied from `Configuration.GDPR.HostVendorID`. The data center is the only new field we need to add to the config, which can just be at the top level IMHO.

Reply:
Awesome, sounds good, will update!